pythonx/coverage_highlight.py

```diff
@@ -510,14 +496,22 @@ def jump_to_next():
     next_range = signs.find_next_range(row)
     if next_range is None:
         current_range = signs.find_current_range(row)
-        if current_range is not None:
-            # if we're at the very last range, jump to the last line
-            first, last = current_range
-            if row != last:
-                vim.command("normal! %dG" % last)
+        if vim.eval("&wrapscan"):
+            next_range = signs.find_first_range()
+            if next_range is None:
+                print("No highlighted lines in buffer")
                 return
-        print("No highlighted lines below cursor")
-        return
+            elif next_range == current_range:
+                next_range = None
```

Comment:
Oooh thank you for considering this corner case!  I would've forgotten to check for it while reviewing.